### ISSUE TITLE
CSRF_TRUSTED_ORIGINS missing in reference configuration.py

### ISSUE THREAD

**Issue by RobinBeismann** [netbox-community/netbox-docker#737]
### Current Behavior

Netbox introduced the parameter "CSRF_TRUSTED_ORIGINS" as required parameter in ``configuration.py`` as Django 4.0 requires the URL Scheme to be set.
The reference ``configuration.py`` does not allow setting this value via the ENV File.

The Netbox Issue for this is: https://github.com/netbox-community/netbox/issues/8509#issue-1121107307

### Expected Behavior

The reference configuration needs to allow setting this variable via the docker envionment variables like
`CSRF_TRUSTED_ORIGINS = list(filter(None, environ.get('CSRF_TRUSTED_ORIGINS', 'https://localhost').split(' ')))`

I'll prepare a pull request in a few minutes.

### Docker Compose Version

1.25.0

### Docker Version

```text
Client: Docker Engine - Community
 Version:           20.10.14
 API version:       1.41
 Go version:        go1.16.15
 Git commit:        a224086
 Built:             Thu Mar 24 01:48:02 2022
 OS/Arch:           linux/amd64
 Context:           default
 Experimental:      true

Server: Docker Engine - Community
 Engine:
  Version:          20.10.14
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.16.15
  Git commit:       87a90dc
  Built:            Thu Mar 24 01:45:53 2022
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.5.11
  GitCommit:        3df54a852345ae127d1fa3092b95168e4a88e2f8
 runc:
  Version:          1.0.3
  GitCommit:        v1.0.3-0-gf46b6ba
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```

### The git Revision

head

### The git Status

```text
On branch main
```

### Startup Command

docker compse up

### NetBox Logs

```text
-
```

### Content of docker-compose.override.yml

```yaml
-
```

**Comment by RobinBeismann** [comment]
Sorry, just read that I shouldn't open an issue when opening a PR. But now that it is here, I'll reference.

**Comment by Fyne5** [comment]
Oh, did you let netbox-docker behind the reverse proxy like as Nginx? I put my Nginx'config here for you or someone want it, of course it works for me: Cloudflare with SSL > Nginx reverse proxy without SSL > Netbox-docker

```
    server {
        listen      8080;
        access_log  off;
        location / {
            proxy_pass http://netbox:8080;
            proxy_set_header X-Forwarded-Host $http_host;
            proxy_set_header X-Real-IP $remote_addr;
            proxy_set_header X-Forwarded-Proto https;
            add_header P3P 'CP="ALL DSP COR PSAa PSDa OUR NOR ONL UNI COM NAV"';
        }
    }
```

**Comment by RobinBeismann** [comment]
> Oh, did you let netbox-docker behind the reverse proxy like as Nginx? I put my Nginx'config here for you or someone want it, of course it works for me: Cloudflare with SSL > Nginx reverse proxy without SSL > Netbox-docker
> 
> ```
>     server {
>         listen      8080;
>         access_log  off;
>         location / {
>             proxy_pass http://netbox:8080;
>             proxy_set_header X-Forwarded-Host $http_host;
>             proxy_set_header X-Real-IP $remote_addr;
>             proxy_set_header X-Forwarded-Proto https;
>             add_header P3P 'CP="ALL DSP COR PSAa PSDa OUR NOR ONL UNI COM NAV"';
>         }
>     }
> ```

Yes. Did you already update to the latest image? That change regarding CSRF came with the latest update based on Django 4.0.
We're running it behind KEMP Load Master, but that should in theory behave the same.

**Comment by Fyne5** [comment]
> > Oh, did you let netbox-docker behind the reverse proxy like as Nginx? I put my Nginx'config here for you or someone want it, of course it works for me: Cloudflare with SSL > Nginx reverse proxy without SSL > Netbox-docker
> > ```
> >     server {
> >         listen      8080;
> >         access_log  off;
> >         location / {
> >             proxy_pass http://netbox:8080;
> >             proxy_set_header X-Forwarded-Host $http_host;
> >             proxy_set_header X-Real-IP $remote_addr;
> >             proxy_set_header X-Forwarded-Proto https;
> >             add_header P3P 'CP="ALL DSP COR PSAa PSDa OUR NOR ONL UNI COM NAV"';
> >         }
> >     }
> > ```
> 
> Yes. Did you already update to the latest image? That change regarding CSRF came with the latest update based on Django 4.0. We're running it behind KEMP Load Master, but that should in theory behave the same.

Yes, I did with newest version (netbox-docker 1.6.1 - 3.2.0) and in my case when wrong nginx config: proxy_set_header X-Forwarded-Proto https;

**Comment by tobiasge** [comment]
Fixed in PR #738
